code/modules/religion/rites.dm

```diff
@@ -255,17 +255,94 @@
 
 // Necro Rites
 
+/datum/religion_rites/create_lesser_lich
+	name = "Create Lesser Lich"
+	desc = "Gives the bound creature a spell granting them the ability to create a lesser phylactery, causing them to become a skeleton and revive on death twice if the phylactery still exists on-station. Be warned, becoming a lesser lich will prevent revivial by any other means."
+	ritual_length = 70 SECONDS
+	ritual_invocations = list("From the depths of the soul pool ...",
+	"... come forth into this being ...",
+	"... grant this servant power ...",
+	"... grant them temporary immortality ...")
+	invoke_msg = "... Grant them the power to become one with necromancy!!"
+	favor_cost = 2250
+//the creature chosen for the rite
+	var/mob/living/lich_to_be
+//the spell
+	var/lichspell = /obj/effect/proc_holder/spell/targeted/lesserlichdom
+
+/datum/religion_rites/create_lesser_lich/perform_rite(mob/living/user, atom/religious_tool)
+	if(!ismovable(religious_tool))
+		to_chat(user,"<span class='warning'>This rite requires a religious device that individuals can be buckled to.</span>")
+		return FALSE
+	var/atom/movable/movable_reltool = religious_tool
+	if(!movable_reltool)
+		return FALSE
+	if(LAZYLEN(movable_reltool.buckled_mobs))
+		for(var/creature in movable_reltool.buckled_mobs)
+			lich_to_be = creature
+		if(!lich_to_be.mind.hasSoul)
+			to_chat(user,"<span class='warning'>[lich_to_be] has no soul, as such this rite would not help them. To empower another, they must be buckled to [movable_reltool].</span>")
+			lich_to_be = null
+			return FALSE
+		for(var/obj/effect/proc_holder/spell/knownspell in lich_to_be.mob_spell_list)
+			if(knownspell.type == lichspell)
+				to_chat(user,"<span class='warning'>You've already empowered [lich_to_be], get them to use the spell granted to them! To empower another, they must be buckled to [movable_reltool].</span>")
+				lich_to_be = null
+				return FALSE
+		to_chat(user,"<span class='warning'>You're going to empower the [lich_to_be] who is buckled on [movable_reltool].</span>")
+		return ..()
+	else
```

Comment:
chain all of this into else if blocks, because they all return

Reply:
little confused on what specifically needs to be else-if here, 294?

Reply:
this is gonna sound really dumb but I dont get what you mean as this is what I'm looking at 
![image](https://user-images.githubusercontent.com/17776299/187729270-76cbad73-c122-43f6-aad7-0120c5459628.png)

do you mean like this, because to me this seems wrong, without even trying to compile or test...
![image](https://user-images.githubusercontent.com/17776299/187729585-0c420433-16e1-4cc6-ac7a-8362e4c80151.png)

Reply:
oh, guh, mobile formatting. sorry. it's good as it is